docs/06_how-to-guides/60_how-to-return-values-from-actions.md

```diff
@@ -69,3 +76,7 @@ The following options are available when you complete the procedure:
 
 [[info | Returned values from actions availability]]
 The returned values from actions are only available to the clients sending the action via the RPC API. Currently, there is no support for an inline action to be able to use the return value, because inline actions don't execute synchronously.
```

Comment:
"The action return values are only available to clients   ... via the RPC API" ?

"Inline actions execute asynchronously so in current versions inline actions cannot use return values" ?